pynamodb/connection/base.py

```diff
@@ -284,7 +284,7 @@ def dispatch(self, operation_name, operation_kwargs):
 
         Raises TableDoesNotExist if the specified table does not exist
         """
-        if operation_name not in [DESCRIBE_TABLE, LIST_TABLES, UPDATE_TABLE, DELETE_TABLE, CREATE_TABLE]:
+        if operation_name not in FREE_OPERATIONS:
             if RETURN_CONSUMED_CAPACITY not in operation_kwargs:
```

Comment:
can combine the two ifs into a single if

Reply:
Done.